exercises/hello-world/src/HelloWorld.hx

```diff
@@ -0,0 +1,9 @@
+package;
+
+// Simply return the string 'Hello, World!'
+
+class HelloWorld {
+  public static function hello() : String {
+    return "";
```

Comment:
Could you probably replcae the empty string by `"replace me"`, such that a student exactly sees where they need to change the code?

Reply:
I really like that suggestion. Makes it clear as day for a first exercise.